src/enigma2/Recordings.cpp

```diff
@@ -424,6 +439,37 @@ PVR_ERROR Recordings::DeleteRecording(const PVR_RECORDING &recinfo)
   return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR Recordings::UndeleteRecording(const PVR_RECORDING& recording)
+{
+  auto recordingEntry = GetRecording(recording.strRecordingId);
+
+  std::regex regex(TRASH_FOLDER);
+  std::string replaceWith = "";
+
+  const std::string newRecordingDirectory = regex_replace(recordingEntry.GetDirectory(), regex, replaceWith);
```

Comment:
Do you really need var `repaceWith`? Can't you just call `regex_replace` with "" as last parameter?

Reply:
Nope